udata_front/theme/gouvfr/datagouv-components/src/locales/fr.json

```
@@ -64,5 +64,9 @@
   "Learn more about this indicator - opens a new window": "En savoir plus sur cet indicateur - ouvre une nouvelle fenêtre",
   "Learn more about this indicator": "En savoir plus sur cet indicateur",
   "Read more": "Lire plus",
-  "Read less": "Lire moins"
+  "Read less": "Lire moins",
+  "Private": "Privé",
+  "Archived": "Archivé",
+  "{n} reuses": "{n} reuses",
```

```suggestion
  "{n} reuses": "{n} réutilisations | {n} réutilisation | {n} réutilisations",
```
All other strings with `{n}` should be updated to provide cases for singular value